### PR TITLE
Create basic  hardcoded results endpoints and frontend

### PR DESCRIPTION
**Pull request by Lev-Stambler**
Closing issue #24

- Create the endpoints which return dummy data
- Create a simple, non-styled frontend which gets data from the backend and displays the results

### CODE REVIEW

File: biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+export const SimulatePage = () => {
+  const [simulationResults, setSimulationResults] = useState<
+    BiogridSimulationResults
+  >();
+  const [simulationRan, setSimulationRan] = useState(false);
+
+  const client = Client.getInstance();
+  async function runSimulation() {
+    await client.api.runBiogridSimulation();
```
Comment (Lev-Stambler) [inline biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx:23]: here is an example of the auto client being used

File: biogrid/apps/api/src/app/controllers/BiogridController.ts [hunk 2]
```diff
@@ -28,4 +25,18 @@ export class BiogridController extends Controller {
   public async NewBiogrid(@Body() body: NewBiogridBody): Promise<void> {
     await createNewBiogrid();
   }
```
Comment (RolandNaijuka) [inline biogrid/apps/api/src/app/controllers/BiogridController.ts]: you get the body, but do not use it anywhere. Could you pass the body to the func ```createNewBiogrid()```

File: biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './simulate-page.css';
+import { Client } from '../../client';
+import { BiogridSimulationResults } from '../../build';
+
+export const SimulatePage = () => {
+  const [simulationResults, setSimulationResults] = useState<
+    BiogridSimulationResults
+  >();
+  const [simulationRan, setSimulationRan] = useState(false);
```
Comment (RolandNaijuka) [inline biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx]: nit: could you use variable names like. ```simulationStatus``` or ```gridStatus``` for ```simulationRan```. This enables easy readability and relates to the project. You can extrapolate the function from the variable.

Comment (Lev-Stambler) [reply biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx]: Changed to an enum!

**Pull request merged**